tests/dummy/app/pods/components/frost-create-form/template.hbs

```diff
@@ -0,0 +1,18 @@
+{{#frost-modal-input title='Test title' subtitle='Subtitle'}}
+{{!#frost-form-container titleComponent='myTitle'}}
+
+  {{frost-bunsen-form
```

Comment:
I don't like that we call it `bunsen` in the component (I mean, how is that significant to anyone consuming this) - maybe we make a transition to just `frost-form` in the future